### DISCUSSION TITLE
Thank you for forking SMT !

### DISCUSSION THREAD

**Discussion by ShogunWeb** [FossifyOrg/General-Discussion#167] [General]
Just a little comment to thank you for forking SMT !
For several months, I tested some "acceptable" open source alternatives to these beloved apps, but was not very happy with what I found. Then finally discovered you forked the SMT apps, and installed all the apps again ! Love it !

So i'm a happy returning user, and also happy be a sponsor (as I was for SMT in the old days). 
Thank you Naveen and all contributors for your dedication, and for making the project alive !!

**Comment by s676p** [comment]
Very nice indeed that SMT lives on ( as Fossify )
One question though - are there any tools / easy ways/ hacks to transfer contacts, calendar, notes, ... to Fossify ? All come up empty when I install the replacement apps. 

peter

**Reply by Aga-C** [reply]
Some of the SMT apps have data export in Settings. Then in Fossify apps you can import it, also in Settings.
